src/pages/register/hooks/useRegisterMutation.ts

```
@@ -0,0 +1,41 @@
+import axiosInstance from '@/api/axiosInstance';
+import { ENDPOINTS } from '@/api/endpoints';
+import { useAuth } from '@/hooks/useAuth';
+import type { RegisterFormInputs } from '@/pages/register/utils/registerValidation';
+
+export interface RegisterResponse {
+  accessToken?: string;
+  refreshToken?: string;
+}
+
+function toPayload(data: RegisterFormInputs) {
+  const { confirmPassword, mbti, phone, ...rest } = data;
```

ESLint

[error] 'confirmPassword' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)